LAYOUT_STYLE_GUIDE.md

```diff
@@ -408,6 +409,41 @@ The text in square brackets will be included as the image's alt text. Similar to
 
 In order to properly add images to a lesson, follow the instructions in our [Adding Images to the Curriculum](https://github.com/TheOdinProject/curriculum/wiki/Adding-Images-to-the-Curriculum) Wiki page to get a statically URL as seen in the codeblock above.
 
+## Keyboard shortcuts
+
+For keyboard shortcuts we use the HTML keyboard input element `<kbd>`.
+
+Example code which will be rendered as: <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>?</kbd>
+
+```html
+<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>?</kbd>
+``` 
+
+### Style standardization
+
+- Use the HTML `<kbd>` tag for individual keys:
```

Comment:
Nit:

```suggestion
- Use separate `<kbd>` elements for individual keys:
```

Basically just to convey the intent of the following code snippet a bit more, i.e. we don't want users doing `<kbd>Ctrl + Shift</kbd>`

Reply:
makes sense, thanks!